koku/reporting/migrations/0319_node_network_costs.py

```diff
@@ -0,0 +1,199 @@
+                ("cluster_alias", models.TextField(null=True)),
+                ("cluster_id", models.TextField()),
+                ("cost_model_cpu_cost", models.DecimalField(decimal_places=15, max_digits=33, null=True)),
+                ("cost_model_memory_cost", models.DecimalField(decimal_places=15, max_digits=33, null=True)),
+                ("cost_model_rate_type", models.TextField(null=True)),
+                ("cost_model_volume_cost", models.DecimalField(decimal_places=15, max_digits=33, null=True)),
+                ("data_source", models.CharField(max_length=64, null=True)),
+                ("distributed_cost", models.DecimalField(decimal_places=15, max_digits=33, null=True)),
+                ("infrastructure_data_in_gigabytes", models.DecimalField(decimal_places=15, max_digits=33, null=True)),
+                (
+                    "infrastructure_data_out_gigabytes",
+                    models.DecimalField(decimal_places=15, max_digits=33, null=True),
+                ),
+                ("infrastructure_markup_cost", models.DecimalField(decimal_places=15, max_digits=33, null=True)),
+                ("infrastructure_raw_cost", models.DecimalField(decimal_places=15, max_digits=33, null=True)),
+                ("node", models.CharField(max_length=253, null=True)),
+                ("raw_currency", models.TextField(null=True)),
+                ("resource_count", models.IntegerField(null=True)),
+                (
+                    "resource_ids",
+                    django.contrib.postgres.fields.ArrayField(
+                        base_field=models.CharField(max_length=256), null=True, size=None
+                    ),
+                ),
+                ("usage_end", models.DateField()),
+                ("usage_start", models.DateField()),
+                (
+                    "cost_category",
+                    models.ForeignKey(
+                        null=True, on_delete=django.db.models.deletion.CASCADE, to="reporting.openshiftcostcategory"
+                    ),
+                ),
+                (
+                    "source_uuid",
+                    models.ForeignKey(
+                        db_column="source_uuid",
+                        null=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="reporting.tenantapiprovider",
+                    ),
+                ),
+            ],
+            options={
+                "db_table": "reporting_ocp_network_summary_by_node_p",
+                "indexes": [
+                    models.Index(fields=["usage_start"], name="ocp_net_summ_node_usage_start"),
+                    models.Index(fields=["node"], name="ocp_net_summ_node_node"),
+                ],
+            },
+        ),
+        migrations.RunSQL(
+            "ALTER TABLE reporting_ocp_network_summary_by_node_p ALTER COLUMN ID SET DEFAULT uuid_generate_v4();",
+            reverse_sql="SELECT 1;",
+        ),
+    ]
```

Comment:
We need to include the unset code on the end of the migration like we run the set code at the top.
```suggestion
        ),
        migrations.RunPython(code=unset_pg_extended_mode, reverse_code=set_pg_extended_mode),
    ]
```

Reply:
Done.